outaTime/outaTime/View Controllers/TimeCircuitsViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  TimeCircuitsViewController.swift
+//  outaTime
+//
+//  Created by Dojo on 6/9/20.
+//  Copyright © 2020 Dojo. All rights reserved.
+//
+
+import UIKit
+
+class TimeCircuitsViewController: UIViewController {
+    
+    @IBOutlet var destinationText: UILabel!
+    @IBOutlet var presentText: UILabel!
+    @IBOutlet var departedText: UILabel!
+    @IBOutlet var speedText: UILabel!
+    
+    @IBOutlet var travelBackButton: UIButton!
+    
+    @IBAction func travelBackTapped(_ sender: UIButton) {
+        startTimer()
+    }
+    
+    let dateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "MMM dd yyyy"
+        formatter.timeZone = TimeZone(secondsFromGMT: 0)
+        return formatter
+    }()
+    
+    var currentSpeed = 0
+    var myTimer: Timer?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let timeNow = Date()
+        presentText?.text = dateFormatter.string(from: timeNow)
+        speedText?.text = "\(currentSpeed) MPH"
+        destinationText?.text = "--- -- ----"
+        departedText?.text = "--- -- ----"
+        
+    }
+    
+    func startTimer() {
+        myTimer = Timer(timeInterval: 0.1, repeats: true, block: updateSpeed(timer:))
+        if let timer = myTimer {
+            RunLoop.current.add(timer, forMode: .default)
+        }
```

Comment:
Very nice job on this timer and all of the logic for updating it. Your code reads nice and clean.